src/pybind/cephfs/cephfs.pyx

```diff
@@ -1096,6 +1097,26 @@ cdef class LibCephFS(object):
         if ret < 0:
             raise make_ex(ret, "error in setxattr")
 
+    def removexattr(self, path, name):
```

Comment:
mind adding a test in `src/test/pybind/test_cephfs.py`?

Reply:
I've added the test, please check it.